tests/integration/test_distributed_directory_monitor_split_batch_on_failure/test.py

```diff
@@ -41,7 +41,7 @@ def test_distributed_directory_monitor_split_batch_on_failure_OFF(started_cluste
         node2.query(f'insert into dist select number/100, number from system.numbers limit {limit} offset {limit*i}', settings={
             # max_memory_usage is the limit for the batch on the remote node
             # (local query should not be affected since 30MB is enough for 100K rows)
-            'max_memory_usage': '30Mi',
+            'max_memory_usage': '25Mi',
```

Comment:
You forget to update `test_distributed_directory_monitor_split_batch_on_failure_ON` too (and a comment above)